auth/services/services.go

```diff
@@ -0,0 +1,92 @@
+package services
+
+import (
+	"github.com/nuts-foundation/go-did"
+	"net/http"
+	"time"
+
+	irma "github.com/privacybydesign/irmago"
+	"github.com/privacybydesign/irmago/server"
+
+	"github.com/nuts-foundation/nuts-node/auth/contract"
+)
+
+// ContractValidator interface must be implemented by contract validators
+// deprecated
+type ContractValidator interface {
+	// ValidateContract validates a signed login contract
+	ValidateContract(contract string, format ContractFormat, checkTime *time.Time) (*ContractValidationResult, error)
+	// ValidateJwt validates a JWT that contains a signed login contract
+	ValidateJwt(contract string, checkTime *time.Time) (*ContractValidationResult, error)
+	IsInitialized() bool
+}
+
+// ContractSessionHandler interface must be implemented by ContractSessionHandlers
+// deprecated
+type ContractSessionHandler interface {
+	SessionStatus(session SessionID) (*SessionStatusResult, error)
+	StartSession(request interface{}, handler server.SessionHandler) (*irma.Qr, string, error)
+}
+
+// OAuthClient is the client interface for the OAuth service
+type OAuthClient interface {
+	CreateAccessToken(request CreateAccessTokenRequest) (*AccessTokenResult, error)
+	CreateJwtBearerToken(request CreateJwtBearerTokenRequest) (*JwtBearerTokenResult, error)
+	IntrospectAccessToken(token string) (*NutsAccessToken, error)
+	Configure() error
+}
+
+// AuthenticationTokenContainerEncoder defines the interface for Authentication Token Containers services
```

Comment:
Is this VP compatible, otherwise add a deprecated comment

Reply:
Wasn't used anymore, removed.